dcc-portal-ui/app/scripts/repository/views/repository.icgc.html

```diff
@@ -113,24 +128,35 @@ <h3>
             </div>
         </div>
         <div class="narrow omega" data-ng-if="ICGCRepoController.downloadEnabled">
-            <div data-ng-if="ICGCRepoController.isSafari && !ICGCRepoController.isChrome" class="alert alert-info" style="margin:0 0 16px;">
-                <a href="#" class="close pull-right" style="cursor:pointer;" data-dismiss="alert" aria-label="close">&times;</a>
+            <div data-ng-if="ICGCRepoController.isSafari && !ICGCRepoController.isChrome" class="alert alert-info"
+                style="margin:0 0 16px;">
+                <a href="#" class="close pull-right" style="cursor:pointer;" data-dismiss="alert"
+                    aria-label="close">&times;</a>
                 <strong>
                     <span>Using Safari browser?</span>
                 </strong>
-                <br> We know that users have problems downloading gzip files from the download server. The downloaded files only
+                <br> We know that users have problems downloading gzip files from the download server. The downloaded
+                files only
                 contain file headers. To fix the problem please either use any other browser or disable the "Open 'safe'
                 files after downloading" option in Safari browser as described in
                 <a href="https://discussions.apple.com/thread/2297632?tstart=0" target="_blank">this article</a>.
             </div>
             <p style="margin-top:0;">
                 <span>
-                    <a href="http://docs.icgc.org/portal/repositories/#having-trouble-downloading-a-file-from-dcc-repository" target="_blank">
+                    <a href="https://docs.icgc.org/download/repositories/" target="_blank">
```

Comment:
All changes are from this line and down. The rest are html auto-formatting from the IDE.